plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/core/CoreResources_it.properties

```diff
@@ -1,69 +1,413 @@
+
+confirm_exit_toggleMessage = Non chiedere ancora
+
+confirm_version_check_message = La nuova versione {1} di {0} \u00E8 stata rilasciata il {2}.\nVuoi aggiornare la tua versione attuale?
 
-confirm_version_check_message = La nuova versione {1} di {0} \u00E8 stata rilasciata il {2}.\n\Vuoi aggiornare la tua versione attuale?
 confirm_version_check_title = Scarica la nuova versione
+
 confirm_version_check_toggleMessage = Non chiedere ancora
 
 controls_itemlist_action_copy = Copia
 
 dialog_about_font = SansSerif
+
 dialog_about_label_version = Versione 
+
 dialog_about_title = About
 
+dialog_connection_advanced_settings = Impostazioni avanzate
+
 dialog_connection_button_test = &Tenta di stabilire una Connessione ...
+
 #page conn setting
 dialog_connection_description = Impostazioni della connessione al Database.
-dialog_connection_advanced_settings = Impostazioni avanzate
+
+dialog_connection_edit_title = Configurazione della connessione "{0}"
+
+dialog_connection_edit_wizard_bad_pwd_msg = La password non corrisponde
+
+dialog_connection_edit_wizard_bad_pwd_title = Password errata
+
+dialog_connection_edit_wizard_conn_change_question = La connessione "{0}" \u00E8 stata modificata.\r\nVuoi connnetterti di nuovo?
+
+dialog_connection_edit_wizard_conn_change_title = Connessione modificata
+
+dialog_connection_edit_wizard_connections = Identificazione del client
+
+dialog_connection_edit_wizard_connections_description = Identificatione del client di connessione
+
+dialog_connection_edit_wizard_data_format = Formato dei dati
+
+dialog_connection_edit_wizard_editors = Editor binario
+
+dialog_connection_edit_wizard_error_md5_msg = Impossibile generare hash della password
+
+dialog_connection_edit_wizard_error_md5_title = Errore nella generazione MD5
+
+dialog_connection_edit_wizard_general = Generale
+
+dialog_connection_edit_wizard_general_bootstrap_query_ignore_error_lable = Ignora errori SQL
+
+dialog_connection_edit_wizard_general_bootstrap_query_sql_label = Query SQL
+
+dialog_connection_edit_wizard_general_bootstrap_query_sql_title = Inserisci SQL
+
+dialog_connection_edit_wizard_lock_pwd_title = Inserisci password di lock
+
+dialog_connection_edit_wizard_metadata = Metadata
+
+dialog_connection_edit_wizard_metadata_description = Preferenze di lettura dei metadati
+
+dialog_connection_edit_wizard_presentation = Presentazione
+
+dialog_connection_edit_wizard_presentation_description = Interfaccia dei risultati e presentazione
+
+dialog_connection_edit_wizard_resultset = Editor dei dati
+
+dialog_connection_edit_wizard_shell_cmd = Comandi Shell
+
+dialog_connection_edit_wizard_shell_cmd_directory_label = Cartella di lavoro
+
+dialog_connection_edit_wizard_shell_cmd_directory_title = Cartella di lavoro
+
+dialog_connection_edit_wizard_shell_cmd_pause_label = Metti l'esecuzione in pausa dopo (ms)
+
+dialog_connection_edit_wizard_shell_cmd_variables_hint_label = Non si possono usare variabili nei comandi. Clicca per vedere l'elenco.
```

Comment:
Hi, @capitanfuturo! Thank you for contribution.
I used translator and noticed that _"Non si possono usare variabili nei comandi"_ means something like _"You **can't** use variables in commands"_, while this property contains "_You **can** use variables in commands"_ in English properties file. Could you, please, check, is this a mistake?

Reply:
Hi @E1izabeth, thank you for your review. Yes it's my mistake, I checked the English version and in Italian it should be "Puoi usare delle variabili nei comandi"

Reply:
Thank you for correction!